src/Graphics/AnimationFlashing.cpp

```diff
@@ -0,0 +1,14 @@
+#include "AnimationFlashing.h"
+
+void AnimationFlashing::run(int64_t millis) {
```

Comment:
Curious about this function, is there an example of where it is used? Any reason it can't be on a timer or something?

Reply:
it's in SteeringDisplay::run where it iterates through the _animations unordered_map.

Rather than having a separate timer for each animation, SteeringDisplay has _animationTimer which is used for all animations.  I thought it was a bit cleaner that way so just a matter of preference I suppose

Reply:
Ohh that makes sense, probably more efficient too!